packages/calico/extra/start-calico-felix.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -xe
+
+export IP=`/opt/mesosphere/bin/detect_ip`
+export NODENAME=`hostname`
+
+# the default configuration path, `/etc/calico/calicoctl.cfg`, of calicoctl is
+# used to simply the interaction with Calico
+/usr/bin/mkdir -p /etc/calico
+/usr/bin/cp /opt/mesosphere/etc/calico/calicoctl.cfg /etc/calico/calicoctl.cfg
+
+/opt/mesosphere/bin/calicoctl apply -f \
+	/opt/mesosphere/active/calico/etc/profile.yaml
+
+# nodename file name is expected to be created before starting up calico-node
+mkdir -p /var/lib/calico
```

Comment:
Could you please store all calico files in `/var/*/dcos/calico`?

Reply:
Sure, but we may have some expcetions:
- `/etc/calico/calicoctl.cfg` is the default path for calicoctl, we keep this path so that we don't have to specify a configuration path for calicoctl
- `/etc/calico/confd` is a hard-coded configuration path for confd.